.github/workflows/ci.yml

```diff
@@ -22,6 +30,7 @@ jobs:
     container:
       image: ${{ inputs.container-image || 'timescaledev/toolkit-builder:debian-11-amd64' }}
     strategy:
+      fail-fast: false
```

Comment:
Is this temporary for debugging?

Reply:
discussed at standup, decided we want to disable fast failure